### DISCUSSION TITLE
Issues with the Contentful App

### DISCUSSION THREAD

**Discussion by igoraugustynski** [swellstores/community#324] [Help] [answered]
Hello,
I've installed and configured the Swell app for Contentful. 
After setting up a Content Model with a field of type "Short text, List" I'm trying to select some products.
<img width="1006" alt="Screenshot 2023-07-27 at 12 08 58" src="https://github.com/swellstores/community/assets/94485687/a5b1c5a8-7738-44de-94c6-b19f908982fa">

The selection itself does work -> I can see all my products and select them, but after I press the "Save Products" button it looks like Contentful does not receive any data -> The "Publish changes" button does not update. Even if I change another field and try to publish the changes, they are not saved (and the field resets after a page reload) and I don't get any data from the Contentful API.
<img width="1430" alt="Screenshot 2023-07-27 at 12 10 13" src="https://github.com/swellstores/community/assets/94485687/294600ff-9f75-4a63-9cbd-ff2a4893e9b3">

**Comment by igoraugustynski** [comment]
I also tried with another field of type "short-text" set to required, and after selecting a product it still shows the "required" error. Looks like no values are retrieved from Swell.
<img width="597" alt="Screenshot 2023-07-27 at 12 12 31" src="https://github.com/swellstores/community/assets/94485687/41e56bb3-0742-488d-8f2f-e13f132c5082">

**Comment by igoraugustynski** [comment]
Anyone?

**Comment by steven-k1** [comment]
Hey @igoraugustynski I'll take a look into this

**Reply by igoraugustynski** [reply]
Any news on this?

**Reply by igoraugustynski** [reply]
How can I use Your system in a production environment if nothing works and You guys just ignore the bugs?

**Comment by steven-k1** [comment, marked as answer]
@igoraugustynski The reported issue is being looked into. From my testing, I have not been able to reproduce what you reported, so I will need to do some additional testing on my end.

**Reply by igoraugustynski** [reply]
Still no news on this one. Were You able to reproduce the issue?
